src/SharedPipelineComponents/Datalock/SFA.DAS.CollectionEarnings.DataLock/DataLockProcessor.cs

```diff
@@ -37,6 +37,7 @@ protected DataLockProcessor()
         public virtual void Process()
         {
             _logger.Info("Started Data Lock Processor.");
+            _logger.Info($"Using timeout of: {DataLockTask.CommandTimeout}");
```

Comment:
is there a reason to hard code the timeout here? (the static CommandTimeout is hardcoded)